### DISCUSSION TITLE
Need to help in using of react router tab

### DISCUSSION THREAD

**Discussion by monster1017** [rpldy/react-uploady#436] [Q&A] [answered]
Hello, everyone.
I made a component called react-router-tabs  using react-router-dom .
But It seems a routing don't work well.
I am looking for someone to help my issue.
Best regards.

**Comment by technexxa** [comment, marked as answer]
Hi, @monster1017.
I saw your article just before.
What is version of react-router-com?

**Comment by monster1017** [comment]
Hi, @Apollo-111.
Thank you for your reply.
Version is 5.0.0.

**Comment by technexxa** [comment]
Oh, It seems that version of react-router-dom isn't compatible with react version.

**Comment by yoavniran** [comment]
@monster1017 @Apollo-111 doesnt look like this discussion belongs here, right? Or is it somehow related to Uploady?

**Comment by monster1017** [comment]
Yeah, I upgraded version of react-router-com to 6.4.4.
It seems a component is going well.
@Apollo-111, thanks for your kind help.
